src/electron/platform/android/test-configs/tab-stops/test-config.tsx

```diff
@@ -1,19 +1,23 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { NeedsReviewInstancesSection } from 'common/components/cards/needs-review-instances-section';
 import { UnifiedFeatureFlags } from 'electron/common/unified-feature-flags';
 import { TestConfig } from 'electron/types/test-config';
 import { VirtualKeyboardView } from 'electron/views/virtual-keyboard/virtual-keyboard-view';
 import * as React from 'react';
+import * as styles from '../../../../views/results/test-view.scss';
+
+const ariaLevelForHowToTestHeading: number = 2;
 
 export const tabStopsTestConfig: TestConfig = {
     key: 'tab-stops',
     contentPageInfo: {
         title: 'Tab stops',
         description: (
-            <>
-                <strong>How to test:</strong>
-                <ol>
+            <div className={styles.tabStopsTest}>
```

Comment:
I'm not sure I see the reason for this to be a div?

Reply:
I changed this back to not being a div, I had made it a div to handle the nested css that I've now removed.